Source/WebCore/platform/graphics/cocoa/CMUtilities.mm

```diff
@@ -277,6 +283,129 @@ static CFStringRef convertToCMYCbCRMatrix(PlatformVideoMatrixCoefficients coeffi
     return adoptCF(rawSampleBuffer);
 }
 
+PacketDurationParser::PacketDurationParser(const AudioInfo& info)
+{
+    AudioStreamBasicDescription asbd { };
+    asbd.mFormatID = info.codecName.value;
+    UInt32 size = sizeof(asbd);
+    auto error = PAL::AudioFormatGetProperty(kAudioFormatProperty_FormatInfo, info.cookieData->size(), info.cookieData->data(), &size, &asbd);
+    if (error || !info.rate) {
+        RELEASE_LOG_ERROR(Media, "createAudioFormatDescription failed with error %d (%.4s)", (int)error, (char*)&error);
+        return;
+    }
+    m_audioFormatID = asbd.mFormatID;
+    m_sampleRate = info.rate;
+    m_constantFramesPerPacket = asbd.mFramesPerPacket;
+#if HAVE(AUDIOFORMATPROPERTY_VARIABLEPACKET_SUPPORTED)
+    switch (m_audioFormatID) {
+#if ENABLE(VORBIS)
+    case kAudioFormatVorbis:
+        {
```

Comment:
Nit: the brace should be on the previous line.

Reply:
webkit-style is so hopeless then. It whinged until I had it setup that way